projects/bp-gallery/src/matomo.ts

```diff
@@ -0,0 +1,20 @@
+const setupMatomo = (url: string) => {
+  const w: any = window;
+  const _paq: Array<any> = (w._paq = w._paq || []);
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
```

Comment:
could you add
```
_paq.push(["disableCookies"]);
```
here? requireCookieConsent may do the same thing but disableCookies may be more explicit